airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -576,6 +596,13 @@ def execute(self, context: Context) -> str | None:
 
         return self.job_id
 
+    def execute_complete(self, context, event=None):
+        if event["status"] != "success":
+            raise AirflowException(f"Error while running job: {event}")
+        else:
+            self.log.info(event["message"])
+            return event["job_id"]
```

Comment:
It seems to me that this else block might not be needed.

```suggestion
        if event["status"] != "success":
            raise AirflowException(f"Error while running job: {event}")

        self.log.info(event["message"])
        return event["job_id"]
```